include/tvm/driver/driver_api.h

```diff
@@ -166,6 +166,23 @@ TVM_DLL runtime::Module build(const Map<Target, IRModule>& input, const Target&
  */
 TVM_DLL runtime::Module build(const Map<String, IRModule>& input, const Target& target_host);
 
+/*!
+ * \brief Creates TIR Buffer for provided parameters
+ * \param shape shape of the buffer
+ * \param dtype data type
+ * \param name buffer name
+ * \param data_alignment alignment requirement of data pointer in bytes
+ * \param offset_factor Factor of elem_offset field, elem_offset is guaranteed to be
+ *                      multiple of offset_factor
+                        User can specify data_alignment and offset_factor to be 0
+ *                      A default value will be picked.
+ * \param compact If the statement has already bound to a compact buffer.
+ * \param memory_scope memory scope of the buffer
+ */
+TVM_DLL tir::Buffer BufferWithOffsetAlignment(Array<PrimExpr> shape, DataType dtype,
```

Comment:
Hey do you think it's possible to move this API to buffer.h?

Reply:
probably make sense, will try to move